app/src/main/res/values/strings.xml

```diff
@@ -191,7 +191,7 @@
     <!-- Browser menu toggle that installs a Progressive Web App shortcut to the site on the device home screen. -->
     <string name="browser_menu_install_on_homescreen">Install</string>
     <!-- Menu option on the toolbar that takes you to synced tabs page-->
-    <string name="synced_tabs">Synced tabs</string>
+    <string name="synced_tabs" moz:removedIn="98" tools:ignore="UnusedResources">Synced tabs</string>
```

Comment:
Just wondering about this removal. Are we removing `view_synced_tabs_title` because it is never used?

Reply:
Yes. Based on some context clues of git annotations, I am under the impression that Sync previously lived under its own screen + screen title, and the layout + string are leftover from that era after synced tabs were moved to the tabs tray.